inst/@sym/adjoint.m

```diff
@@ -25,10 +25,11 @@
 %% @example
 %% @group
 %% A = [sym(1); 2];
-%% y =  adjoint(A)
+%% y = adjoint(A)
```

Comment:
its possible avoid this line for https://github.com/cbm755/octsympy/pull/536?
or you can merge it and then i update the pr.